// 抛出异常
// const drive = () => {
//     throw new Error ('没油了');
// }

// drive ();

// try...catch...

// const getGasoline = () => {
//     return false;
// }

// const drive = () => {
//     const gasoline = getGasoline();
//     if (!gasoline) {
//         throw new Error ('没油了') ;
//     }
//     console.log ('呜呜~');
// }

// try {
//     drive ();
// } catch (error) {
//     console.log(error.message);
// }
